client/app/components/composites/SearchBar/SearchBar.css

```diff
@@ -390,4 +322,28 @@
   @media (--large-viewport) {
     display: none;
   }
+
+}
+
+/* CSS triangle pointing to the search toggle icon */
+.mobileToggleArrow {
+  display: none;
+  position: absolute;
+  bottom: -1px;
+  right: calc(var(--SearchBar_mobileHeight) / 2 - var(--SearchBar_mobileTriangleSize));
+  border: var(--SearchBar_mobileTriangleSize) solid transparent;
+
+  /* We must adjust the brightness here to match either the input
+     background or the submit button background when the toggle
+     position changes in the breakpoint. This cannot be done in JS by
+     switching the color since the breakpoints are unknown there. */
+  filter: brightness(80%);
```

Comment:
No wait, I cannot really do this. The reason why the filter is in the CSS is that it changes based on the viewport width.

On a narrow screen the arrow is next to the darker submit button:

<img width="479" alt="screen shot 2016-08-09 at 10 14 50" src="https://cloud.githubusercontent.com/assets/53923/17507752/27c9f6e4-5e1a-11e6-94b3-20864619140b.png">

but on a wider screen the arrow is next to the lighter input background:

<img width="663" alt="screen shot 2016-08-09 at 10 18 17" src="https://cloud.githubusercontent.com/assets/53923/17507842/a189f132-5e1a-11e6-9b48-329c70c68454.png">

And since JS isn't aware of the breakpoints we cannot change the color dynamically without a resize handler that checks the current breakpoint. And I'd like to avoid doing that.